object_detection/pytorch/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Code of Conduct
+
+Facebook has adopted a Code of Conduct that we expect project participants to adhere to.
```

Comment:
I'm fine with this, but someone should likely take a look if this is okay to put in the MLPerf repo